backoffice/src/wwwroot/js/accordion.js

```diff
@@ -0,0 +1,20 @@
+
+export function initializeAccordion() {
+
+    var acc = document.getElementById("triageHelp");
```

Comment:
I wonder if there's a better way to do this, 

something more re-usable like a js-accordion class 

then loop through all the elements with this and add the click listener